webapp/packages/core-data-context/src/DataContext/DataContextGetter.ts

```diff
@@ -5,9 +5,11 @@
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
-import type { IDataContextProvider } from './IDataContextProvider';
+
+const typescriptTypeLink = Symbol('typescript type link');
```

Comment:
Looks strange. Is there a way to avoid this?

Reply:
I don't know any way to keep the generic type without using it in the object